src/main/scala/nl/knaw/dans/easy/license/app/LicenseCreatorService.scala

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.license.app
+
+import java.io.{ByteArrayOutputStream, File, FileOutputStream}
+import javax.naming.Context
+import javax.naming.ldap.InitialLdapContext
+
+import com.yourmediashelf.fedora.client.{FedoraClient, FedoraCredentials}
+import nl.knaw.dans.easy.license.LicenseCreator
+import nl.knaw.dans.easy.license.internal.Parameters
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+import org.eclipse.jetty.ajp.Ajp13SocketConnector
+import org.eclipse.jetty.server.Server
+import org.scalatra.servlet.ScalatraListener
+import org.eclipse.jetty.servlet.ServletContextHandler
+import org.scalatra.{InternalServerError, Ok, ScalatraServlet}
+import rx.schedulers.Schedulers
+import nl.knaw.dans.easy.license.internal._
+
+import scala.util.Try
+
+class LicenseCreatorService extends ApplicationSettings with DebugEnhancedLogging {
+  import logger._
+
+  private val port = props.getInt("daemon.http.port")
+  val server = new Server(port)
+  val context = new ServletContextHandler(ServletContextHandler.NO_SESSIONS)
+  context.addEventListener(new ScalatraListener())
+
+  server.setHandler(context)
+  info(s"HTTP port is $port")
+
+  if (props.containsKey("daemon.ajp.port")) {
+    val ajp = new Ajp13SocketConnector()
```

Comment:
I don't think it merits a large comment. If you wan't to know what AJP is, you can google it. In any case, that sort of thing should become basic knowledge for all developers.